src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -295,6 +295,9 @@
     <value>Open the terminal with the provided title instead of the profile's set "title"</value>
     <comment>{Locked="\"title\""}</comment>
   </data>
+  <data name="CmdTabColorArgDesc" xml:space="preserve">
+    <value>Open the terminal with the tab color, in #rrggbb format</value>

```

Comment:
```suggestion
    <value>Open this tab with the specified tab color, in #rrggbb format</value>
```

(since this is an argument to nt/new-tab you can refer to an implicit tab. Panes technically.. won't quite work like this (!))